pyo3-build-config/src/impl_.rs

```diff
@@ -155,6 +157,24 @@ impl InterpreterConfig {
         for flag in &self.build_flags.0 {
             println!("cargo:rustc-cfg=py_sys_config=\"{}\"", flag)
         }
+
+        if cfg!(feature = "export-config") {
```

Comment:
I think this might get called by all crates which call `pyo3_build_config::use_pyo3_cfgs`. Might be better to just put this in the PyO3 build script?

Reply:
Sure - that makes sense to me. The only case I can think of would be another crate basically substituting for PyO3  but also using `pyo3-build-config`. Does that make sense?